test/testenv/appframework_utils.go

```diff
@@ -311,3 +328,162 @@ func WaitforPhaseChange(deployment *Deployment, testenvInstance *TestEnv, name s
 		time.Sleep(1 * time.Second)
 	}
 }
+
+// Verifications will perform several verifications needed between the different steps of App Framework tests
+func Verifications(deployment *Deployment, testenvInstance *TestEnv, appSource []AppSourceInfo, appVersion string, splunkPodAge map[string]time.Time, status string, clusterManagerBundleHash string, scaling string) string {
+	/* Function Steps
+	 * Verify apps 'download' state for all CRs
+	 * Verify apps 'podCopy' state for all CRs
+	 * Verify apps packages are deleted from the operator pod for all CRs
+	 * Verify apps 'install' state for all CRs
+	 * Verify apps packages are deleted from the CR pods
+	 * Verify bundle push is successful
+	 * Verify apps are copied to correct location on CR pods
+	 * Verify no pods did reset
+	 * Verify apps are installed to correct location on CR pods
+	 */
+	// Verify apps 'Download' and 'PodCopy' states for all CRs
+	phases := []enterpriseApi.AppPhaseType{enterpriseApi.PhaseDownload, enterpriseApi.PhasePodCopy}
+	for _, phase := range phases {
+		for _, appSource := range appSource {
+			testenvInstance.Log.Info(fmt.Sprintf("Verify apps '%v' state on %v CR", phase, appSource.CrKind))
+			VerifyAppListPhase(deployment, testenvInstance, appSource.CrName, appSource.CrKind, appSource.CrAppSourceName, phase, appSource.CrAppFileList)
+		}
+	}
+
+	// Verify apps packages are deleted from the operator pod for all CRs
+	opPod := GetOperatorPodName(testenvInstance.GetName())
+	for _, appSource := range appSource {
+		testenvInstance.Log.Info(fmt.Sprintf("Verify apps %s packages are deleted from the operator pod for %v", appVersion, appSource.CrKind))
+		opPath := filepath.Join(splcommon.AppDownloadVolume, "downloadedApps", testenvInstance.GetName(), appSource.CrKind, deployment.GetName(), appSource.CrAppScope, appSource.CrAppSourceName)
+		VerifyAppsPackageDeletedOnContainer(deployment, testenvInstance, testenvInstance.GetName(), []string{opPod}, appSource.CrAppFileList, opPath)
+	}
+
+	// Verify apps 'install' state for all CRs
+	phases = []enterpriseApi.AppPhaseType{enterpriseApi.PhaseInstall}
+	phase := phases[0]
+	for _, appSource := range appSource {
+		testenvInstance.Log.Info(fmt.Sprintf("Verify apps '%v' state on %v CR", phase, appSource.CrKind))
```

Comment:
Replace `testenvInstance.Log.Info(fmt.Sprintf("Verify apps '%v' state on %v CR", phase, appSource.CrKind))` with `testenvInstance.Log.Info(fmt.Sprintf("Verify apps '%v' state on %v CR", enterpriseApi.PhaseInstall, appSource.CrKind))`
Same for `VerifyAppListPhase` replace `phase` with `enterpriseApi.PhaseInstall`
Then we can remove 
`phases = []enterpriseApi.AppPhaseType{enterpriseApi.PhaseInstall}`
`phase := phases[0]`